app/models/public_body.rb

```diff
@@ -102,7 +102,7 @@ class ImportCSVDryRun < StandardError ; end
   :eager_load => [:translations]
   has_tag_string
 
-  strip_attributes :allow_empty => false
+  strip_attributes :allow_empty => false, :except => [:request_email]
```

Comment:
This should get squashed in to https://github.com/mysociety/alaveteli/pull/4084/commits/d8268fda2b5603a98351582020d6b5df7d787d83